documentation/style.css

```diff
@@ -156,7 +157,7 @@ p {
 	color: var(--dark-gray);
 }
 
-@media screen and (max-width: 700px) {
+@media screen and (max-width: 800px) {
 	.logo {
 		display: none;

```

Comment:
which screen are your targeting here by mentioning 800px?

Reply:
For a breakpoint almost at 800px, the main content was pushed behind the sidebar and was not properly seen, so I thought to make 800px as a breakpoint. Should I change it?

Reply:
if its the case of ipad you are handling 768px might also work

Reply:
Updated.